content/docs/latest/installing/cloud/digitalocean.md

```diff
@@ -127,7 +127,7 @@ read SSH_KEY_ID
 If not done yet, [create a custom image](https://developers.digitalocean.com/documentation/v2/#create-a-custom-image) from the current Flatcar Container Linux Stable version:
 
 ```shell
-VER=$(curl https://stable.release.flatcar-linux.net/amd64-usr/current/version.txt | grep -m 1 FLATCAR_VERSION_ID= | cut -d = -f 2)
+VER=$(source <(curl https://stable.release.flatcar-linux.net/amd64-usr/current/version.txt); echo "${FLATCAR_VERSION_ID}")
```

Comment:
If this should serve as reference for copy paste it would be good to have error reporting with `set -euo pipefail` still work. The `<(CMD)` usage swallows the error - maybe an explicit check for a non-empty string could be added before the echo.